tests/unit/responses/HttpResponse.test.ts

```diff
@@ -1,7 +1,7 @@
 import 'mocha';
 import assert from 'assert';
 
-import HttpResponse from '../../../lib/responses/HttpResponse';
+import { HttpResponse } from '../../util';

```

Comment:
I'm not a fan of you testing utility class, which extends actual unit. This might lead to confusion as for example I didn't understand where does restoreDefaultHeaders method comes from. Test actual HttpResponse.